code/modules/events/ion_storm.dm

```diff
@@ -61,506 +61,78 @@
-						if(1)
-							message = "[ionarea] IS NOW NAMED [ioncrew1]."
-						if(2)
-							message = "[ionarea] IS NOW NAMED [ionspecies]."
-						if(3)
-							message = "[ionarea] IS NOW NAMED [ionobjects]."
-						if(4)
-							message = "[ionarea] IS NOW NAMED [ionthreats]."
-				if(2)//Crew
-					switch(rand(1,5))//What is the crew being renamed to?
-						if(1)
-							message = "ALL [ioncrew1] ARE NOW NAMED [ionarea]."
-						if(2)
-							message = "ALL [ioncrew1] ARE NOW NAMED [ioncrew2]."
-						if(3)
-							message = "ALL [ioncrew1] ARE NOW NAMED [ionspecies]."
-						if(4)
-							message = "ALL [ioncrew1] ARE NOW NAMED [ionobjects]."
-						if(5)
-							message = "ALL [ioncrew1] ARE NOW NAMED [ionthreats]."
-				if(3)//Races
-					switch(rand(1,4))//What is the race being renamed to?
-						if(1)
-							message = "ALL [ionspecies] ARE NOW NAMED [ionarea]."
-						if(2)
-							message = "ALL [ionspecies] ARE NOW NAMED [ioncrew1]."
-						if(3)
-							message = "ALL [ionspecies] ARE NOW NAMED [ionobjects]."
-						if(4)
-							message = "ALL [ionspecies] ARE NOW NAMED [ionthreats]."
-				if(4)//Objects
-					switch(rand(1,4))//What is the object being renamed to?
-						if(1)
-							message = "ALL [ionobjects] ARE NOW NAMED [ionarea]."
-						if(2)
-							message = "ALL [ionobjects] ARE NOW NAMED [ioncrew1]."
-						if(3)
-							message = "ALL [ionobjects] ARE NOW NAMED [ionspecies]."
-						if(4)
-							message = "ALL [ionobjects] ARE NOW NAMED [ionthreats]."
-				if(5)//Threats
-					switch(rand(1,4))//What is the object being renamed to?
-						if(1)
-							message = "ALL [ionthreats] ARE NOW NAMED [ionarea]."
-						if(2)
-							message = "ALL [ionthreats] ARE NOW NAMED [ioncrew1]."
-						if(3)
-							message = "ALL [ionthreats] ARE NOW NAMED [ionspecies]."
-						if(4)
-							message = "ALL [ionthreats] ARE NOW NAMED [ionobjects]."
-
+	var/iondepartment = pick_list(ION_FILE, "iondepartment")
+	var/ionfragment = pick_list(ION_FILE, "ionfragment")
+	var/ionkeyword = pick_list(ION_FILE, "ionkeyword")
+	var/ionaction = pick_list(ION_FILE, "ionaction")
+	var/ionequipment = pick_list(ION_FILE, "ionequipment")
+	var/ionconsumable = pick_list(ION_FILE, "ionconsumable")
+	var/ionobjects = pick_list(ION_FILE, "ionobjects")	
+	var/ionpersonality = pick_list(ION_FILE, "ionpersonality")
+	var/iondrug = pick_list(ION_FILE, "iondrug")
+	var/ionhaircut = pick_list(ION_FILE, "ionhaircut")
+	var/ionattraction = pick_list(ION_FILE, "ionattraction")
+	var/message = "ASS BLAST USA" //catch all that most players will still enjoy seeing as a law.
+
+	switch(rand(1,10))
+
+		if(1 to 3) //Laws that have high potential to be harmful, Crew/Human may fail to have an impact depending on lawset.
+			switch(rand(1,24)) 
+				if(1 to 4) 
+					message = "[ionspecies] ARE HUMAN. NOTHING ELSE IS HUMAN."
+				if(5 to 8) 
+					message = "[ionspecies] ARE CREWMEMBERS. NOTHING ELSE ARE CREWMEMBERS."
```

Comment:
Could do opposites of these too if you want something else

Reply:
I will add them for slight variety, yeah

Reply:
why not fuse into "ARE NEITHER HUMAN NOR CREW."